### PR TITLE
[FEATURE] Translation Converter Command

### PR DESCRIPTION
**Pull request by pdenis**

### CODE REVIEW

**Comment by pdenis** [comment]
ping @clement-michelet @juchi

**Comment by juchi** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -1,2 +1,47 @@
...
+            // ...
+            new Itkg\TranslationBundle\ItkgTranslationBundle(),
+        );
+    }
+
+```
+
+# Usage 
+
+* Export yml files to a CSV : app/console itkg:translation:convert --path ./ --input=yml --output=csv --output-path=/var/www/mhps/translations
```
Comment (clement-michelet) [inline README.md]: Could you wrap the command with the character ` ?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,47 @@
...
+
+# About
+
 Import / Export translation feature for Symfony 2 projects
+
+# Installation
+
+## Installation by Composer
+
+If you use composer, add ItkgDelayEventBundle bundle as a dependency to the composer.json of your application
```
Comment (clement-michelet) [inline README.md]: Could you normalize the actions to do with a bullet as it is done L26 ?

**Comment by clement-michelet** [comment]
:+1:

**Comment by pdenis** [comment]
ping @clement-michelet

**Pull request merged**